sentencepiece/Rakefile

```diff
@@ -78,5 +89,60 @@ class SentencePiecePackageTask < PackageTask
   end
 end
 
+class SentencePieceRepositoryTask < RepositoryTask
+  def initialize(packages_red_data_tools_repository, groonga_repository)
+    @packages_red_data_tools_repository = packages_red_data_tools_repository
+    @groonga_repository = groonga_repository
+  end
+
+  def define
+    super
+    Rake::Task["apt:upload"].clear
+    define_apt_upload_task
+  end
+
+  def gpg_uid
+    File.read(File.join(@groonga_repository,
+                        "gpg_uid")).chomp
+  end
+
+  def rsync_base_path
+    "packages@packages.groonga.org:public"
+  end
+
+  def gpg_key_path
+    File.join(@groonga_repository,
+              "packages",
+              "yum",
+              "RPM-GPG-KEY-groonga")
+  end
+
+  def repositories_dir
+    "apt/repositories"
+  end
+
+  def distributions
+    ["buster"]
+  end
+
+  def define_apt_upload_task
+    namespace :apt do
+      desc "Upload repositories"
+      task :upload => [repositories_dir] do
+        distributions.each do |distribution|
+          sh("rsync",
+             "-avz",
+             "--progress",
+             "--delete",
+             "#{repositories_dir}/#{distribution}/",
+             "#{rsync_base_path}/#{distribution}")
+        end
+      end
+    end
+  end
+end
+
 task = SentencePiecePackageTask.new(apache_arrow_repository)
```

Comment:
Could you also use `groonga_repository` to find `upload.rb`?

Reply:
Thanks, I've fixed.